runner/src/main.rs

```diff
@@ -509,11 +509,11 @@ fn run_example_server(
             // TODO(#396): Add `--oidc-client` support.
             format!("--application={}", application_file),
             match opt.server_variant {
-                ServerVariant::Base => format!("--permissions={}", permissions_file),
+                ServerVariant::Base | ServerVariant::Experimental => format!("--permissions={}", permissions_file),
```

Comment:
Does it work? Experimental uses the `oak_unsafe` feature:
https://github.com/project-oak/oak/blob/87632c651af99ff49815e1da35185c1341d97e80/runner/src/main.rs#L323

But the `permissions` option is only available when `oak_unsafe` is disabled.

Reply:
hm,
Didn't know `experimental` used `oak_unsafe`
Last time I checked - it didn't

Reply:
Does it mean - we shouldn't use permissions with `experimental` at all ?

Reply:
I guess you can remove `oak_unsafe` from `experimental`. I added it when I made the default set of features empty (before that `oak_unsafe` was included by default). If you remove `oak_unsafe`, I think you have to remove `oak_introspection_client` too. I think `experimental` should work without `oak_unsafe` if you don't need introspection. 

(Actually, I think `oak_introspection_client` does not make sense without `oak_unsafe`. So, maybe https://github.com/project-oak/oak/blob/87632c651af99ff49815e1da35185c1341d97e80/oak_runtime/Cargo.toml#L15 should be change; possibly in a separate PR.)

Reply:
I remember it didn't work without `oak_introspection_client`, but I need to re-check